javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -165,6 +165,16 @@ public void shouldTuple8ReturnNotNull() {
         assertThat(tuple(1, 2, 3, 4, 5, 6, 7, 8)).isNotNull();
     }
 
+    @Test
+    public void shouldRightReturnNotNull() {
+        assertThat(right(null)).isNotNull();
```

Comment:
isn't this the same as if I would have imported the method from `Either`?
@danieldietrich, what's the take on that, is this meant as a shortcut for only importing the `API`?